### PR TITLE
Release 2.3.10

### PR DESCRIPTION
**Pull request by finnlewis**
Update patches based on recent upstream changes to work with Drupal 9.4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,32 +7,65 @@ This project is the Drupal installation profile that is best installed using
...
-## Requirements for installing LocalGov Drupal locally
+## Requirements for installing LocalGov Drupal locally for testing and development
+
+To install LocalGov Drupal locally you will need an appropriate versions of:
+
+ - PHP (see https://www.drupal.org/docs/system-requirements/php-requirements)
+ - A database server like MySQL (see https://www.drupal.org/docs/system-requirements/database-server-requirements)
+ - A web server like APache2 (see https://www.drupal.org/docs/system-requirements/web-server-requirements) 
+
+Many of us use the Lando file included to run a local docker environmnent for testing and development, but seom people prefer to run the web servers natively on their host machine.
```
Comment (Adnan-cds) [inline README.md:31]: s/seom/some/ :)

File: README.md [hunk 2]
```diff
@@ -7,32 +7,65 @@ This project is the Drupal installation profile that is best installed using
...
+
+### PHP requirements
 
-To install locally, you will need Composer and we recommend Lando.
+We folllow Drupal's PHP recomendations: https://www.drupal.org/docs/system-requirements/php-requirements#versions
```
Comment (Adnan-cds) [inline README.md:35]: s/folllow/follow/

File: README.md [hunk 3]
```diff
@@ -7,32 +7,65 @@ This project is the Drupal installation profile that is best installed using
...
 
-https://getcomposer.org/
-https://lando.dev/
+We currently recomend PHP 8.1 or 8.0 but also aim to support PHP 7.4.
```
Comment (Adnan-cds) [inline README.md:37]: s/recomend/recommend/

**Review by Adnan-cds — APPROVED** [review]
There are a few typos, but otherwise looks okay.  Thanks :)

**Review by stephen-cox — APPROVED** [review]
Let's get this release out and fix the typos on another issues :+1:

**Pull request merged**